data-model/fdo-type/virtual-collection/0.1.0/schema/virtual-collection.json

```diff
@@ -5,180 +5,114 @@
+                1,
+                2,
+                3
             ]
         },
-        "dcterms:creator": {
-            "type": "object",
-            "description": "The creator of the collection.",
-            "properties": {
-                "ods:type": {
-                    "type": "string",
-                    "description": "The type of creator of the collection.",
-                    "examples": [
-                        "foaf:Person"
-                    ],
-                    "enum": [
-                        "foaf:Person",
-                        "foaf:Organisation",
-                        "oa:SoftwareAgent"
-                    ],
-                    "required": [
-                        "ods:id",
-                        "ods:type"
-                    ]
-                },
-                "foaf:name": {
-                    "type": "string",
-                    "description": "The name of the creator. https://xmlns.com/foaf/0.1/#term_name.",
-                    "examples": [
-                        "My Name"
-                    ]
-                }
-            }
-        },
-        "ltc:ObjectGroup": {
-            "type": "object",
-            "description": "https://tdwg.github.io/ltc/quick-reference/#ObjectGroup An intentionally grouped set of objects with one or more common characteristics.",
-            "properties": {
-                "ltc:conditionsOfAccess": {
-                    "type": "string",
-                    "description": " https://tdwg.github.io/ltc/terms/index.html#ObjectGroup_conditionsOfAccess Information about who can access the collection being described or an indication of its security status.",
-                    "examples": [
-                        "Open to the public"
-                    ]
-                },
-                "ltc:collectionName": {
-                    "type": "string",
-                    "description": "https://tdwg.github.io/ltc/terms/index.html#ObjectGroup_collectionName A short title that summarizes the collection objects contained within the ObjectGroup.",
-                    "examples": [
-                        "Arctiacaja1991Netherlands"
-                    ]
-                },
-                "ltc:description": {
-                    "type": "string",
-                    "description": "https://tdwg.github.io/ltc/terms/index.html#ObjectGroup_description A free text description or narrative about the collection.",
-                    "examples": [
-                        "A virtual collection representing all butterfly specimens in South Holland"
-                    ]
-                }
-            }
+        "ltc:collectionName": {
+            "type": "string",
+            "description": "A short title that summarizes the collection objects contained in the virtual collection.",
+            "examples": [
+                "Bumblebee Collection of South Holland"
+            ]
         },
-        "ltc:MeasurementOrFact": {
```

Comment:
Do not know what to fill here. I would rather not keep a count of the amount of records in the collection as with each ingestion this could change. We can easily do this dynamically if required.

Reply:
Separate request object that contains this. In the storage we will not use it